apps/marketing/src/components/PostHog.astro

```diff
@@ -0,0 +1,57 @@
+---
+import { env } from '@orbitkit/env/marketing';
+---
+
+<script define:vars={{ env }} is:inline>
+  !(function (t, e) {
+    var o, n, p, r;
+    e.__SV ||
+      ((window.posthog = e),
+      (e._i = []),
+      (e.init = function (i, s, a) {
+        function g(t, e) {
+          var o = e.split('.');
+          2 == o.length && ((t = t[o[0]]), (e = o[1])),
+            (t[e] = function () {
+              t.push([e].concat(Array.prototype.slice.call(arguments, 0)));
+            });
+        }
+        ((p = t.createElement('script')).type = 'text/javascript'),
+          (p.async = !0),
+          (p.src = s.api_host + '/static/array.js'),
+          (r = t.getElementsByTagName('script')[0]).parentNode.insertBefore(
+            p,
+            r,
+          );
+        var u = e;
+        for (
+          void 0 !== a ? (u = e[a] = []) : (a = 'posthog'),
+            u.people = u.people || [],
+            u.toString = function (t) {
+              var e = 'posthog';
+              return (
+                'posthog' !== a && (e += '.' + a), t || (e += ' (stub)'), e
+              );
```

Comment:
what is this block of code? I looked over posthog's docs about astro and I couldn't see this.

Reply:
<img width="578" alt="Screenshot 2024-06-08 at 1 51 37 PM" src="https://github.com/ixahmedxi/orbitkit/assets/23265119/563ab83c-c65d-4786-85d6-7a98ed8dfb26">

Reply:
Can we include a comment with a reference to this documentation? I myself would be weirded out by minified code not knowing what it does.

Reply:
> Can we include a comment with a reference to this documentation? I myself would be weirded out by minified code not knowing what it does.

I agree. A comment with a reference to the documentation would be nice in this case or maybe we can make it work in another (better?) way.
https://github.com/fogbender/b2b-saaskit/blob/c6fa937bbd80d1ef745cd08fa0b213e8420d65a7/src/components/head/posthog.ts